idea/jvm-debugger/jvm-debugger-test/test/org/jetbrains/kotlin/idea/debugger/test/util/KotlinOutputChecker.kt

```diff
@@ -75,7 +77,20 @@ internal class KotlinOutputChecker(
             LOG.error("Test file created ${outFile.path}\n**************** Don't forget to put it into VCS! *******************")
         } else {
             val originalText = FileUtilRt.loadFile(outFile, CharsetToolkit.UTF8)
-            val expected = StringUtilRt.convertLineSeparators(originalText)
+            val ignoreDirectives = mutableListOf<TargetBackend>()
+            val expected = StringUtilRt.convertLineSeparators(originalText).split("\n").filter {
+                if (it.trim().startsWith("// IGNORE_BACKEND: ")) {
+                    val backends = it.substringAfter(":").split(",")
+                    backends.forEach {
+                        val trimmed = it.trim()
+                        if (trimmed == "JVM_IR") ignoreDirectives.add(TargetBackend.JVM_IR)
```

Comment:
Maybe `ignoreDirecrives.add(TargetBackend.valueOf(trimmed))`?

Reply:
Gone in current version.